docker/reqmgr2ms-unmerged/slc7-cernonly.repo

```diff
@@ -0,0 +1,9 @@
+[slc7-cernonly]
+name=CentOS-7 - CERN Only [20190324]
```

Comment:
I don't know where you get it but it seems very outdated, e.g. cmsweb uses `name=CentOS-7 - CERN Only [20200928]`, and as such it is newer. I suggest that you take whatever latest from lxplus.

Reply:
this is a copy  from `reqmgr2ms` - nothing new. I am going to change it to the one that  is used in `cmsweb` though

Reply:
end-to-end deployment approach implies that you manage your images as you like. If reqmgr2ms is outdated it is still DMWM group issue, I just pointed to outdated version based on provided PR.

Reply:
done